src/index.ts

```diff
@@ -50,6 +50,11 @@ server.get(`/auth/realms/${config.realm}/protocol/openid-connect/auth`, async (r
             padding: .5rem;
             border-radius: .25rem;
           }
+
+          li > div {
```

Comment:
@mariusheine Not directly related, but could you check if the mobile view works atm?

Reply:
@anbraten I checked it via the devtools and there it looked totally fine

![image](https://github.com/geprog/oauth-mock-server/assets/42845535/10ed83d8-9936-4c91-97b4-94b9ec7e2a8c)

Reply:
great. might be fixed by Lukas already, but we probably did not updated it